charts/airbyte-pod-sweeper/templates/deployment.yaml

```diff
@@ -44,6 +44,29 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        # Values from secret
+        {{- if .Values.secrets }}
+        {{- range $k, $v := .Values.secrets }}
+        - name: {{ $k }}
+          valueFrom:
+            secretKeyRef:
+              name: server-secrets
+              key: {{ $k }}
+        {{- end }}
+        {{- end }}
+
+        # Values from env
+        {{- if .Values.env_vars }}
+        {{- range $k, $v := mergeOverwrite .Values.env_vars .Values.global.env_vars }}
```

Comment:
does this expansion language have `STDOUT` I feel like tracking this variables is pretty challenging, it would be nice to see what we are overwriting